addon/helpers/icons.js

```diff
@@ -3,6 +3,8 @@ export const ICONS = {
   add: { default: 'add' },
   arrowBottom: { default: 'arrowDown' },
   arrowLeft: { default: 'arrowLeft' },
+  arrowMenuClose: { default: 'arrowMenuClose', plainIcon: 'arrowMenuClose_plain' },
+  arrowMenuOpen: { default: 'arrowMenuOpen', plainIcon: 'arrowMenuOpen_plain' },
```

Comment:
On peut enlever les plainIcon car elle sont à celles pas plain/ par défaut 😄 

```suggestion
  arrowMenuClose: { default: 'arrowMenuClose' },
  arrowMenuOpen: { default: 'arrowMenuOpen' },
```